config-templates/salesforce/leads.yaml

```diff
@@ -0,0 +1,71 @@
+data_source:
+  name: salesforce_lead
+  owners:
+    - support@transformdata.io
+  sql_table: salesforce.lead ### Update to your leads schema.table
+  mutability:
+    type: full_mutation
+    type_params:
+      update_cron: 0 10 * * *
+  identifiers:
+    - name: lead_id
```

Comment:
Same as `account` comment - change this to `lead` so that when referenced in the query it's `lead__name` etc.